### DISCUSSION TITLE
I can't find the GIS folder, where is it?

### DISCUSSION THREAD

**Discussion by volm92** [InterDigitalInc/AdvantEDGE#257] [Q&A] [answered]
[Here](https://github.com/InterDigitalInc/AdvantEDGE/wiki/overview-gis) it is written how to put the maps for the GIS feature.
The problem is that I can't find the folder `<your-installation-folder>/.meep/omt`. 
Maybe I misinterpret <your-installation-folder>?
I'd like to know what <your-installation-folder> means, thanks!

**Comment by dilallkx** [comment, marked as answer]
Hi @volm92,

`your-installation-folder` should be your home directory. You should have a `.meep` folder in your home directory.

This wiki page provides common platform installation locations:
https://github.com/InterDigitalInc/AdvantEDGE/wiki/mgmt-cheat-sheet#deployment-files--paths

Thanks!

**Reply by volm92** [reply]
Ohhh I was very close! I was looking for it inside the advantedge folder. Sorry for the trouble, it was such a simple thing!
